### PR TITLE
Update milestone link in readme

### PR DESCRIPTION
**Pull request by stealthybox**
On the road to v1.10 ✅ :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,11 +13,11 @@ You can reach the maintainers of this project at:
...
 
-## Roadmap for v1.6
+## Roadmap for v1.10
 
 The full picture of which direction we're taking is described in [this blog post](http://blog.kubernetes.io/2017/01/stronger-foundation-for-creating-and-managing-kubernetes-clusters.html).
 
-Please also refer to the the [v1.6 milestone in this repo](https://github.com/kubernetes/kubeadm/milestone/1).
+Please also refer to the the [v1.10 milestone in this repo](https://github.com/kubernetes/kubeadm/milestone/5).
 
 ### Code of conduct
```
Comment (luxas) [inline README.md]: Please remove this, as we have a dedicated file for this, referencing the official CoC and as most other repos don't link to this file in the main README either.

File: README.md [hunk 2]
```diff
@@ -13,11 +13,11 @@ You can reach the maintainers of this project at:
 - Weekly Zoom Meetings: https://zoom.us/j/166836624 every Tuesday from 17:00 to 18:00 UTC+0
   - Meeting Notes: https://docs.google.com/document/d/1deJYPIF4LmhGjDVaqrswErIrV7mtwJgovtLnPCDxP7U
 
-## Roadmap for v1.6
+## Roadmap for v1.10
 
 The full picture of which direction we're taking is described in [this blog post](http://blog.kubernetes.io/2017/01/stronger-foundation-for-creating-and-managing-kubernetes-clusters.html).
```
Comment (luxas) [inline README.md]: Please link to kubernetes/community/sig-cluster-lifecycle here

Also there is a new blog post: http://blog.kubernetes.io/2017/10/kubeadm-v18-released.html

**Comment by kargakis** [comment]
I think providing just the open milestones (https://github.com/kubernetes/kubeadm/pull/667) is a better solution, otherwise the README will need to be updated in every release.

**Comment by luxas** [comment]
Merged https://github.com/kubernetes/kubeadm/pull/667 so closing this

**Pull request closed without merging**